test/presentation/recherche/evenement_emploi/actions_recherche_evenement_emploi_view_model_test.dart

```diff
@@ -64,6 +64,17 @@ void main() {
       // Then
       expect(viewModel.withFiltreButton, isTrue);
     });
+
+    test('when recherche status is success and result is empty should return true', () {
```

Comment:
je me dis que le nom est double mensongé : "should return true" alors qu'on teste "isFalse". Et aussi, c'est pas le viewmodel entier qui retourne ça mais une property => "shoud hide filtre button" peut-être ?